README.md

```diff
@@ -1,25 +1,51 @@
 
-# Tsunami agent  
-An implementation of [Osorlab Agent](https://pypi.org/project/ostorlab/) for the [Tsunami scanner](https://github.com/google/tsunami-security-scanner) by Google.  
+<h1 align="center">Agent Tsunami</h1>
+
+<p align="center">
+<img src="https://img.shields.io/badge/License-Apache_2.0-brightgreen.svg">
+<img src="https://img.shields.io/github/languages/top/ostorlab/agent_tsunami">
+<img src="https://img.shields.io/github/stars/ostorlab/agent_tsunami">
+<img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg">
+</p>
+
+_Tsunami is a general purpose network security scanner with an extensible plugin system for detecting high severity vulnerabilities with high confidence._
+
+---
+
+<p align="center">
+<img src="./agent_tsunami.png" alt="agent-tsunami" />
+</p>
+
+An implementation of [Ostorlab Agent](https://pypi.org/project/ostorlab/) for the [Tsunami Scanner](https://github.com/google/tsunami-security-scanner) by Google.
```

Comment:
What is an implementation of what is described?

I suggest adding: This repo is xxx.